service/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.23.0 FATAL_ERROR)
+
+set(PROJECT service)
+project(${PROJECT})
+
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+if(NOT IOS AND NOT ANDROID)
+    #include(common.cmake)
```

Comment:
As I understand it, this is no longer needed, everything works without building the library